src/test/java/org/opensearch/security/dlic/dlsfls/AbstractDlsFlsTest.java

```diff
@@ -54,6 +68,53 @@ protected final void setup(Settings override, DynamicSecurityConfig dynamicSecur
 
         rh = nonSslRestHelper();
     }
+    
+    protected SearchResponse executeSearch(String indexName, String user, String password) throws Exception {
+		HttpResponse response = rh.executeGetRequest("/"+indexName+"/_search?from=0&size=50&pretty",
+				encodeBasicHeader(user, password));
+		Assert.assertEquals(200, response.getStatusCode());
+		XContentParser xcp = XContentType.JSON.xContent().createParser(NamedXContentRegistry.EMPTY,
+				LoggingDeprecationHandler.INSTANCE, response.getBody());
+		return SearchResponse.fromXContent(xcp);
+    }
 
+    protected GetResponse executeGet(String indexName, String id, String user, String password) throws Exception {
+		HttpResponse response = rh.executeGetRequest("/"+indexName+"/_doc/"+id, encodeBasicHeader(user, password));
+		XContentParser xcp = XContentType.JSON.xContent().createParser(NamedXContentRegistry.EMPTY,
```

Comment:
Assert status code here?

Reply:
Depending on the query this might result in different status codes.